resistbot-node-backend/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import { handleIncomingDM, handleQuickResponse, handleRapidProResponse } from './dmHandlers';
+import logger from 'morgan';
+
+var app = express();
+const port = process.env.PORT || 5555;
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+/* GET home page. */
+app.get('/', function(req: express.Request, res: express.Response) {
```

Comment:
In our more "live" app we're gonna need a health endpoint that the orchestrator can ping. Maybe rename this to /health. If we want to ensure connectivity to backend things (discord?) we could ping those in here as well.

Reply:
Changed it to a health check endpoint that I cribbed from the internet!